### PR TITLE
Static shape LLM pipeline out-of-the-box

### PR DESCRIPTION
**Pull request by TolyaTalamanov**

### CODE REVIEW

**Comment by slyalin** [comment]
The transformation was merged. Please recheck.

**Comment by Wovchena** [comment]
You need to update the links to the nightly archive to get it for C++.

File: src/cpp/src/llm_pipeline_static.cpp [hunk 1]
```diff
@@ -89,11 +91,31 @@ void copy_with_left_offset(const ov::Tensor& orig, ov::Tensor& padded) {
...
+			{ "NPUW_ONLINE_AVOID", "P:RMSNorm/NPU" }
+        };
+        stage_cfg.insert(prefill_config.begin(), prefill_config.end());
+    } else if (config_name == "GENERATE_CONFIG") {
+        std::map<std::string, std::string> generate_config = {
+            { "NPU_USE_NPUW", "YES" },
+            { "NPUW_FOLD", "YES" },
+            { "NPUW_DCOFF_TYPE", "f16" },
+            { "NPUW_DCOFF_SCALE", "YES" },
+            { "NPU_COMPILATION_MODE_PARAMS", "compute-layers-with-higher-precision=Sqrt,Power,ReduceMean,Add" },
```
Comment (smirnov-alexey) [inline src/cpp/src/llm_pipeline_static.cpp:114]: `"compute-layers-with-higher-precision=Sqrt,Power,ReduceMean,Add_RMSNorm"`

File: src/cpp/src/llm_pipeline_static.cpp [hunk 2]
```diff
@@ -89,11 +91,31 @@ void copy_with_left_offset(const ov::Tensor& orig, ov::Tensor& padded) {
...
+            { "NPUW_PARALLEL_COMPILE", "YES" },
```
Comment (smirnov-alexey) [inline src/cpp/src/llm_pipeline_static.cpp:115]: Not sure about this one

File: samples/cpp/chat_sample/chat_sample.cpp [hunk 3]
```diff
@@ -4,14 +4,14 @@
 #include "openvino/genai/llm_pipeline.hpp"
 
 int main(int argc, char* argv[]) try {
-    if (2 != argc) {
-        throw std::runtime_error(std::string{"Usage: "} + argv[0] + " <MODEL_DIR>");
+    if (2 != argc || 3 != argc) {
```
Comment (TolyaTalamanov) [inline samples/cpp/chat_sample/chat_sample.cpp:7]: Will return it back until chat is supported

File: src/cpp/src/llm_pipeline_static.cpp (hunk 1 above)
Comment (TolyaTalamanov) [reply src/cpp/src/llm_pipeline_static.cpp]: Done, thanks!

**Review by Wovchena — APPROVED** [review]

**Pull request merged**